tspng/extraction.py

```diff
@@ -139,8 +143,30 @@ def extract_from_folder(path: str, mime_type: str=MIME_TYPE) -> Dict:
         root_ext = os.path.splitext(file)
         if root_ext[1] == '.png':
             # print path name of selected files
-            file_list.append(os.path.join(path, file))
+            file_list.append(path+'/'+file)
```

Comment:
I changed this because the test for `extract_from_folder` would fail on my computer, I can change it back to `os.join.path` and modify the test instead.